### ISSUE TITLE
Partly-cloudy line icons use open cloud

### ISSUE THREAD

**Issue by marcelstoer** [basmilius/weather-icons#30]
The partly-cloudy line icons use a cloud symbol that is not closed at the bottom i.e. like those for the rain/hail/etc. clouds.

Examples:

- https://github.com/basmilius/weather-icons/blob/dev/production/line/svg/partly-cloudy-day.svg
- https://github.com/basmilius/weather-icons/blob/dev/production/line/svg/partly-cloudy-night.svg

**Comment by DadiBit** [comment]
I had noticed the same issue in the past, and I believe this is due to the fact that the node script reduces the svg size by removing also some closing tags, which, I believe, should work fine on browser, but on android (for example) it just breaks the last element.

**Comment by basmilius** [comment]
Hi, sorry for the late answer. Do you mean that the first row in the screenshow should have a closed cloud? Currently working on v3 and I took over the open variant of v2, but I'm open to change it.

<img width="404" alt="image" src="https://github.com/basmilius/weather-icons/assets/978257/2086a01f-abc3-4963-b167-ae4b7809fb74">

**Comment by marcelstoer** [comment]
> Do you mean that the first column in the screenshot should have a closed cloud?

Yes, I think that's what I meant. They look really odd to me. I feel that unless there's precipitation the cloud icon should be closed.

**Comment by basmilius** [comment]
@marcelstoer Thanks for the clarification. I agree :)
